### PR TITLE
fix: Am-309 Gatling Testing: Default data insertion

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-309

### Change description ###
fix: Gatling Testing: Default data insertion(AM-309)

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/amapi/Application.java [hunk 1]
```diff
@@ -1,35 +1,18 @@
...
-    private DefaultRoleSetupImportService importerService;
-
     public static void main(final String[] args) {
         SpringApplication.run(Application.class, args);
     }
 
     @Override
     public void run(String... args) {
-        importerService.addService("cmc");
-        importerService.addResourceDefinition(ResourceDefinition.builder()
```
Comment (bmcts) [inline src/main/java/uk/gov/hmcts/reform/amapi/Application.java]: If the run method is empty, we can remove it and the "implements CommandLineRunner" as well

**Review by bmcts — CHANGES_REQUESTED** [review]
Looks good to me, is it tested on Windows and Mac?

File: src/gatling/testdata/initial-data-load.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+insert into services (service_name, service_description) values ('cmc', null)
+on conflict on constraint services_pkey do update set service_description = null;
```
Comment (shashank-rastogi) [inline src/gatling/testdata/initial-data-load.sql]: any reason for updating service_description = null on conflict; ideally on conflict it should do nothing

File: src/gatling/testdata/initial-data-load.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+insert into resources (service_name, resource_type, resource_name) values ('cmc', 'case', 'claim')
+ on conflict on constraint resources_pkey do nothing;
+
+insert into roles (role_name, role_type, security_classification, access_type) values ('caseworker', cast('RESOURCE' as role_type),cast('PUBLIC' as security_classification), cast('ROLE_BASED' as access_type))
+ on conflict on constraint roles_pkey do update set role_type =cast('RESOURCE' as role_type), security_classification = cast('PUBLIC' as security_classification), access_type = cast('ROLE_BASED' as access_type);
+
+insert into roles (role_name, role_type, security_classification, access_type) values ('Role 1', cast('RESOURCE' as role_type),cast('PUBLIC' as security_classification), cast('ROLE_BASED' as access_type))
```
Comment (shashank-rastogi) [inline src/gatling/testdata/initial-data-load.sql]: Again, on conflict, the query shud do nothing

**Review by shashank-rastogi — CHANGES_REQUESTED** [review]

File: src/gatling/testdata/initial-data-load.sql (hunk 2 above)
Comment (abhijitdiwan) [reply src/gatling/testdata/initial-data-load.sql]: yes right

Comment (abhijitdiwan) [reply src/gatling/testdata/initial-data-load.sql]: done

File: src/main/java/uk/gov/hmcts/reform/amapi/Application.java (hunk 1 above)
Comment (abhijitdiwan) [reply src/main/java/uk/gov/hmcts/reform/amapi/Application.java]: done

**Review by shashank-rastogi — APPROVED** [review]

**Review by bmcts — APPROVED** [review]

**Pull request merged**